lib/parsoid.js

```diff
@@ -104,12 +104,15 @@ class ParsoidService {
 
     _initOpts(opts = {}) {
         this.options = opts;
-        this.parsoidHost = opts.parsoidHost;
+        this.parsoidUri = opts.host || opts.parsoidHost;
         this.options.stash_ratelimit = opts.stash_ratelimit || 5;
-        this.options.grace_ttl = opts.grace_ttl || 86400;
         this._blacklist = compileReRenderBlacklist(opts.rerenderBlacklist);
-        if (!opts.parsoidHost) {
-            throw new Error('Parsoid module: the option parsoidHost must be provided!');
+        if (!this.parsoidUri) {
+            throw new Error('Parsoid module: the option host must be provided!');
+        }
+        // remove the trailing slash, if any
+        if (this.parsoidUri.slice(-1) === '/') {
+            this.parsoidUri = this.parsoidUri.slice(0, -1);
```

Comment:
Why?

Reply:
Because, as the comment two lines above says, we want to get rid of the last slash in case it is provided since we count on it not being there in `_getParsoidReq()`.